modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/pages/Applications.tsx

```diff
@@ -50,6 +51,15 @@ const Applications = () => {
 		setContentModal(ModalType.insurance);
 	};
 
+	const callProducts = (selectCard: string) => {
+		if (cardSelected === insuranceCards[0] || selectCard === 'Personal') {
+			return <ProductCardPersona />;
+		}
+		else {
+			return <ProductCardBusiness />;
+		}
+	};
+
```

Comment:
Please, make the change below.
```suggestion
	const callProducts = (selectCard: string) => {
		if (cardSelected === insuranceCards[0] || selectCard === 'Personal') {
			return <ProductCardPersona />;
		}
		
		return <ProductCardBusiness />;	
	};

```

Reply:
Done!